src/hdmf/container.py

```diff
@@ -43,7 +43,11 @@ def link_resources(self, **kwargs):
         self._external_resources = kwargs['external_resources']
 
     def get_linked_resources(self):
-        return self._external_resources if hasattr(self, "_external_resources") else None
+        if hasattr(self, "_external_resources"):
+            return self._external_resources
+        else:
+            msg = "Cannot find linked ExternalResources."
+            raise ValueError(msg)
```

Comment:
Since this is a get method, I think simply returning None is the more common approach and then a user would check ``if container.get_linked_resources() is None``

Reply:
I'll revert the change.